CMakeLists.txt

```diff
@@ -221,14 +221,19 @@ else()
 
 endif()  # !MSVC
 
-# By default prefer STATIC build (legacy behavior)
-option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
-# only expose shared option to advanced users:
+option(HWY_FORCE_STATIC_LIBS "Ignore BUILD_SHARED_LIBS" OFF)
+option(BUILD_SHARED_LIBS "Build shared libraries" ON)
+# only expose shared/static options to advanced users:
+mark_as_advanced(HWY_FORCE_STATIC_LIBS)
 mark_as_advanced(BUILD_SHARED_LIBS)
 # Define visibility settings globally:
 set(CMAKE_CXX_VISIBILITY_PRESET hidden)
 set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
 
+if (HWY_FORCE_STATIC_LIBS)
+  set(BUILD_SHARED_LIBS OFF)
```

Comment:
How does this interact with the parent project when we're including this as a subdirectory?
Is it scoped to the current directory?

Reply:
The change, IIUC, is local - we do not pass `CACHE BOOL 'Description' FORCE` mantra to `set`

Reply:
Going to investigate if there is a way to emulate BUILD_SHARED_LIBS effects without modifying it...

Reply:
I see, it seems reasonable to override this locally, but let's at least change the default value to OFF in line 224, as it was originally?

Reply:
Done